### DISCUSSION TITLE
A remapper for Mousehook Canary

### DISCUSSION THREAD

**Discussion by VerticusDragoon** [xenia-canary/game-patches#211] [Trash]
I am glad that Xenia exists it makes playing the one True GE64 remaster possible and my monkey brain can't comprehend the complexity commitment that goes into this project so for that I am eternal in my gratefulness...  On the other hand I still can't simply remap controls even if I load up Xenia.  I guess this is probably hard to do or it would have already happened... Right?  So now my actual request...
Would someone out of the kindness of their heart do a build of Mouse-HookCanary:Xenia with a simple onscreen input editor ?  For those of us not at all familiar with ini?  I have tried repeatedly to remap the crouch to shift and walk slow to ctrl to no avail...  I must be overlooking something but I dunno what. 

Any help would be cool too,  but I figured it wasn't going to hurt anything to ask nicely...

**Comment by Margen67** [comment]
Ask \@emoose: https://github.com/emoose/xenia
